lib/components/segmented-control/base.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { KeyboardEvent, RefObject, useEffect, useState } from "react";
+import { TRegularSizes } from "../../types";
+import { Segment } from "./segment";
+import "./segmented-control.scss";
+
+interface Option {
+    disabled?: boolean;
+    icon?: string | React.ReactNode;
+    label?: string;
```

Comment:
`label` can be` string | ReactNode `as some project might have translation for label.

Reply:
thank you) changed to `ReactNode` as its type includes `string` :)